### DISCUSSION TITLE
Setup Multiple Warehouse

### DISCUSSION THREAD

**Discussion by Aledcode** [GreaterWMS/GreaterWMS#234] [Announcements]
Hi, I would like to setup multiple warehouse on this system, the error messages appear like this below image.
please help

![image](https://user-images.githubusercontent.com/110827613/190389919-d3589c43-45e4-4615-9d87-450a020aa4ed.png)

**Comment by Singosgu** [comment]
One Admin is match only one warehouse , If you want Multiple , you should custermize by yourself

**Reply by Aledcode** [reply]
It means 1 Warehouse Name only able to have 1 Admin for current system? 
or 1 warehosue can have more than 1 admin?

**Comment by Singosgu** [comment]
it can ,but you should customize use appid of admin to manage all openid

**Comment by Singosgu** [comment]
it support Multiple Warehouse already
